app.py

```diff
@@ -122,5 +150,17 @@ def post_new_user():
     except Exception as err:
         return f"Internal Server Error", 500
 
+
+@app.route("/users/<email>", methods=["DELETE"])
+def delete_user_email(email):
```

Comment:
not a use case right? I'm assuming for testing purposes?

Reply:
for testing purposes but also probably will have a delete account option in the app so we don't have to make a delete request everytime